### DISCUSSION TITLE
WAINLUX K6 driver

### DISCUSSION THREAD

**Discussion by lessmarble** [arkypita/LaserGRBL#1508] [General]
just got a wainlux k6 and cannot connect.  
can someone make a driver for it and laserRGBL?

**Comment by gmmanonymus111** [comment]
I'm afraid this engraver uses other firmware than GRBL (might be a third party one, developed for this engraver only), which makes it incompatible with LaserGRBL. Your best bet would be aksing the manufacturer/retailer to help you set up your engraver with LaserGRBL.

**Reply by donnels** [reply]
Does anyone know what architecture the wainlux is based on?

**Comment by lessmarble** [comment]
thanks..
